optimize/backend/src/it/java/io/camunda/optimize/test/optimize/VariablesClient.java

```diff
@@ -109,56 +105,56 @@ public List<String> getProcessVariableValues(final ProcessVariableValueRequestDt
         .executeAndReturnList(String.class, Response.Status.OK.getStatusCode());
   }
 
-  public List<String> getProcessVariableValues(
-      final ProcessDefinitionEngineDto processDefinition, final String variableName) {
-    return getProcessVariableValues(null, processDefinition, variableName, STRING);
-  }
-
-  public List<String> getProcessVariableValues(
-      final String processInstanceId,
-      final ProcessDefinitionEngineDto processDefinition,
-      final String variableName) {
-    return getProcessVariableValues(processInstanceId, processDefinition, variableName, STRING);
-  }
-
-  public List<String> getProcessVariableValues(
-      final ProcessDefinitionEngineDto processDefinition,
-      final String variableName,
-      final VariableType variableType) {
-    return getProcessVariableValues(null, processDefinition, variableName, variableType);
-  }
-
-  public List<String> getProcessVariableValues(
-      final String processInstanceId,
-      final ProcessDefinitionEngineDto processDefinition,
-      final String variableName,
-      final VariableType variableType) {
-    ProcessVariableValueRequestDto requestDto = new ProcessVariableValueRequestDto();
-    requestDto.setProcessInstanceId(processInstanceId);
-    requestDto.setProcessDefinitionKey(processDefinition.getKey());
-    requestDto.setProcessDefinitionVersion(processDefinition.getVersionAsString());
-    requestDto.setName(variableName);
-    requestDto.setType(variableType);
-    return getProcessVariableValues(requestDto);
-  }
-
-  public VariableDto createObjectVariableDto(
-      final boolean isNativeJsonVar, final Map<String, Object> variable) {
-    if (isNativeJsonVar) {
-      return createNativeJsonVariableDto(variable);
-    } else {
-      return createMapJsonObjectVariableDto(variable);
-    }
-  }
-
-  public VariableDto createObjectVariableDto(
-      final boolean isNativeJsonVar, final List<Object> variable) {
-    if (isNativeJsonVar) {
-      return createNativeJsonVariableDto(variable);
-    } else {
-      return createListJsonObjectVariableDto(variable);
-    }
-  }
+  //  public List<String> getProcessVariableValues(
```

Comment:
do all these commented methods also need removing?

Reply:
Hm badly commented again, when removing `ProcessDefinitionEngineDto` I commented out helpers used in already commented out IT that required this dto, since we intended to keep helpers as reference for migration. But looking at all these I think the logic is pretty self explanatory from the method names so we really dont need to keep them. Removing

Reply:
Ah, I did actually not forget to add a comment, but its at the top of the class for some reason `// TODO recreate C8 IT equivalent helpers of these with #13337` -- irrelevant now but good to know I wasnt imagining things